README.rdoc

```diff
@@ -0,0 +1,28 @@
+== README
```

Comment:
I don't think it's a requirement of this project, but it can be helpful to replace the info in your READMEs to provide info about your project